src/Whiteboard02_insertShiftArrayTest.java

```diff
@@ -0,0 +1,27 @@
+import static org.junit.Assert.*;
+
```

Comment:
Your function `insertShiftArray` should be in its own file as a class, not in your test file.  Please restructure! :)